.github/workflows/build.yml

```diff
@@ -48,6 +43,12 @@ jobs:
             - name: Lock Symfony version
               run: VERSION=${{ matrix.symfony-version }} .github/workflows/lock-symfony-version.sh
 
+            - name: Remove Symfony proxy-manager-bridge
```

Comment:
Removing this in the job for Symfony 7 hides the fact that this will still be broken for users of the package.

Removing packages in some CI jobs is an acceptable workaround for dev requirements, but not for normal requirements.

My suggestion here would be to drop support for Symfony 5.4 as well (keeping those users on the previous minor version of the extension) as Symfony 6.2+ does not need the proxy-manager-bridge to actually make services lazy.